Makefile

```diff
@@ -24,6 +25,9 @@ protobuf:
 	bash internal/tfplugin5/generate.sh
 	bash plans/internal/planproto/generate.sh
 
+fmt:
+	gofmt -w $(GOFMT_FILES)
```

Comment:
Any reason to not use `go fmt ./...`?

Reply:
True! Not anymore! That var is from the past version of the Makefile when `vendor/` was a thing, thanks for flagging, will update.